src/quacc/runners/ase.py

```diff
@@ -362,3 +369,171 @@ def _set_sella_kwargs(self, optimizer_kwargs: dict[str, Any]) -> None:
 
         if not self.atoms.pbc.any() and "internal" not in optimizer_kwargs:
             optimizer_kwargs["internal"] = True
+
+
+def run_neb(
+    images,
+    relax_cell: bool = False,
+    fmax: float = 0.01,
+    max_steps: int | None = 1000,
+    optimizer: NEBOptimizer | BFGS = NEBOptimizer,
```

Comment:
Is it true that only `NEBOptimizer` or `BFGS` can be used? That doesn't seem to be the case based on the docs. Looks like there are examples with `FIRE` for instance. The only thing I see in the docs is that `BFGSLineSearch` can't be used. Maybe you should just do `NEBOptimizer | Optimizer` and then raise a `ValueError` if the user passes an incompatible one (i.e. `BFGSLineSearch`)?.

Reply:
I tried GPMin and it gave RuntimeError. I have not tried others yet.

Reply:
I have made the requested change though.